src/transforms.js

```diff
@@ -88,6 +101,11 @@ function addDiagnosticKeys(item, options, callback) {
     diagnostic.is_anonymous = true;
   }
 
+  if (item._isUncaught) {
+    diagnostic.is_uncaught = item._isUncaught;
```

Comment:
does this need to be a boolean?

if so we should probably "shebang" it with
```
diagnostic.is_uncaught = !!item._isUncaught;
```

Reply:
This key is diagnostic only and is never evaluated, so shouldn't be an issue.